### DISCUSSION TITLE
怎么解决qq扫码登录异常问题

### DISCUSSION THREAD

**Discussion by Enclaw** [Mrs4s/go-cqhttp#1502] [Q&A]
我的地址是武汉的，今天创建了一个上海的linux服务器，然后qq扫码登录的时候显示登录环境非常用或存在危险，查了一下好像是最近才有的问题，请问一下各位怎么解决
试了把windows的divise.json上传到云，但是没用

**Comment by GKCommander** [comment]
在Windows上成功登入后，把device.json和session.token上传到相应位置后，可以去config.yml里面把密码也给加上，这样应该就不用再扫linux服务器上的码了
